index.bs

```diff
@@ -132,27 +318,39 @@ spec: webidl
       "device-info",
       "background-sync",
       "bluetooth",
+      "persistent-storage",
     };
   </pre>
   <p>
     Each enumeration value in the {{PermissionName}} enum identifies a
-    <dfn export>permission</dfn>, which consists of the following
+    powerful feature. Each powerful feature has the following permission-related
     algorithms and types:
   </p>
   <dl>
     <dt>
       A <dfn export>permission descriptor type</dfn>
     </dt>
     <dd>
-      {{PermissionDescriptor}} or one of its subtypes.
-      If unspecified, this defaults to {{PermissionDescriptor}}.
+      <p>
+        {{PermissionDescriptor}} or one of its subtypes.
+        If unspecified, this defaults to {{PermissionDescriptor}}.
+      </p>
+      <p>
+        The feature can define a partial order on instances. If |descriptorA| is
+        <dfn for="PermissionDescriptor">stronger</dfn> than |descriptorB|, then
+        if |descriptorA|'s <a>permission state</a> is {{"granted"}},
+        |descriptorB|'s <a>permission state</a> must also be {{"granted"}}, and
```

Comment:
Nit: These are statements of fact, so please use "is" instead of "must". for instance, `<a>permission state</a> is {{"granted"}}`.